tla-io/src/test/scala/at/forsyte/apalache/io/quint/TestQuintEx.scala

```diff
@@ -439,4 +452,53 @@ class TestQuintEx extends AnyFunSuite {
   test("can convert builtin assert operator") {
     assert(convert(Q.app("assert", Q.nIsGreaterThanZero)) == "n > 0")
   }
+
+  test("can convert builtin Map operator") {
+    assert(convert(Q.app("Map", Q.intTup1, Q.intTup2)) == "Apalache!SetAsFun({<<0, 1>>, <<3, 42>>})")
+  }
+
+  test("can convert builtin get operator") {
+    assert(convert(Q.app("get", Q.intMap, Q._0)) == "Apalache!SetAsFun({<<0, 1>>, <<3, 42>>})[0]")
+  }
+
+  test("can convert builtin keys operator") {
+    assert(convert(Q.app("keys", Q.intMap)) == "DOMAIN Apalache!SetAsFun({<<0, 1>>, <<3, 42>>})")
+  }
+
+  test("can convert builtin setToMap operator") {
+    assert(convert(Q.app("setToMap", Q.intPairSet)) == "Apalache!SetAsFun({<<1, 2>>, <<1, 2>>})")
+  }
+
+  test("can convert builtin setOfMaps operator") {
+    assert(convert(Q.app("setOfMaps", Q.intSet, Q.intSet)) == "[{1, 2, 3} → {1, 2, 3}]")
+  }
+
+  test("can convert builtin set operator") {
+    assert(
+        convert(Q.app("set", Q.intMap, Q._1, Q._42))
+          ==
+            "[Apalache!SetAsFun({<<0, 1>>, <<3, 42>>}) EXCEPT ![1] = 42]"
```

Comment:
This is fine, because it's just checking syntactic translation, but in principle `set/3` is only defined if `key` is already bound in `map`. Should we change it to:

```suggestion
        convert(Q.app("set", Q.intMap, Q._3, Q._42))
          ==
            "[Apalache!SetAsFun({<<0, 1>>, <<3, 42>>}) EXCEPT ![3] = 42]"
```

Reply:
Ah yeah, mistake on my part. Thanks!